configuration/things.md

```diff
@@ -64,15 +64,23 @@ See the [configuration tutorial]({{base}}/tutorials/beginner/configuration.html)
 Things can also be defined manually by creating `.things` configuration text files.
 These files are stored in `$OPENHAB_CONF/things`.
 
-Benefits of defining Things, Items and other aspects of openHAB in configuration text files are, that they are statically defined, unambiguous, flexible and easy to backup and restore.
-The main downsides of configuration files are the effort needed to compose them and the probability for typing errors.
-
 The syntax for `.things` files is defined as follows (parts in `<..>` are required):
 
 ```xtend
 Thing <binding_id>:<type_id>:<thing_id> "Label" @ "Location" [ <parameters> ]
 ```
 
+The first keyword defines whether the entry is a bridge or a thing. 
+The next statement defines the UID of the thing which contains of the following three segments: binding id, thing type id, thing id. 
+So the first two segments must match to thing type supported by a binding (e.g. `network:device` or `astro:moon`), whereas the thing id can be freely defined. 
+Optionally, you may provide a label in order to recognize it easily, otherwise the default label from the thing type will be displayed.
+
+To help organizing your things, you also may define a location (here: `Location`).
```

Comment:
```suggestion
To help organizing your things, you also may define a location (`Location` in the example above).
```

Reply:
Agreed on all three changes!
I'll change the spaces in the example after this one.

Signed-off-by: Juergen Baginski opus42@gmx.de (github: JueBag)